src/validateCommitMessage.test.js

```diff
@@ -174,8 +176,16 @@ describe('description validation', () => {
   });
 
   it('accepts valid subjects', async () => {
-    await validatePrTitle('fix: foobar', {
+    await validateCommitMessage('fix: foobar', {
       subjectPattern: '^(?![A-Z]).+$'
     });
   });
 });
+
+it('throws an appropriate error message for single commit message errors', async () => {
+  await expect(
+    validateCommitMessage('Fix bug', undefined, 'single commit message')
+  ).rejects.toThrow(
+    'No release type found in single commit message "Fix bug".'
```

Comment:
Currently the full error message looks like this:

> No release type found in single commit message "Validate commit message for 1 commit PRs (and test option)". Add a prefix to indicate what kind of release this pull request corresponds to (see https://www.conventionalcommits.org/).

I'm not sure this helps the PR author to address the issue since we say "Add a prefix". 

Maybe we should clarify that we're working around an issue of Github here, possibly referencing https://github.community/t/how-to-change-the-default-squash-merge-commit-message/1155 and telling the user to add another commit to avoid the issue? That's also what semantic-pull-requests does.

Since the error message becomes a bit different here, maybe it makes sense to pass in an enum value to `validateCommitMessage` that can be identified and used to alter the message as necessary.

E.g.
```js
validateMessage('foo', {type: 'title'})
validateMessage('foo', {type: 'commit'})
```

I've also renamed to function again here, since it can be either a commit message or a PR title.

What do you think?

Reply:
I don't want to complicate the logic in `validateMessage` too much - how about I revert the changes I made there, and use a `try/catch` in `index.js` to replace any single commit error with a catch-all message for the problem?

Reply:
That's a great idea!